packages/forklift-console-plugin/src/modules/Plans/views/details/components/SettingsSection/components/PVCNameTemplate/utils/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const pvcNameTemplateAllowedVariables = [
+  '- .VmName: name of the VM',
+  '- .PlanName: name of the migration plan',
+  '- .DiskIndex: initial volume index of the disk',
+  '- .RootDiskIndex: index of the root disk',
+];
+export const pvcNameTemplateHelperExamples = [
+  '"{{.VmName}}-disk-{{.DiskIndex}}"',
+  '"{{if eq .DiskIndex .RootDiskIndex}}root{{else}}data{{end}}-{{.DiskIndex}}"',
+];
```

Comment:
trans?

Reply:
adding to variables, the example patterns makes no sense to add those since it's should be only in English the yaml